easybuild/easyblocks/m/maple.py

```diff
@@ -35,61 +35,48 @@
 import os
 import shutil
 
-from easybuild.framework.easyblock import EasyBlock
+from easybuild.easyblocks.generic.binary import Binary
 from easybuild.tools.run import run_cmd_qa
 
 
-class EB_Maple(EasyBlock):
+class EB_Maple(Binary):
     """Support for installing Maple."""
 
-    def extract_step(self):
-        """Unpacking of files is just copying Maple binary installer to build dir."""
-
-        for f in self.src:
-            shutil.copy(f['path'], os.path.join(self.builddir, f['name']))
-            f['finalpath'] = self.builddir
-
-    def configure_step(self):
-        """No configuration needed, binary installer"""
-        pass
-
-    def build_step(self):
-        """No compilation needed, binary installer"""
-        pass
-
     def install_step(self):
         """Interactive install of Maple."""
 
         cmd = "%s/Maple%sLinuxX86_64Installer.bin" % (self.builddir, self.cfg['version'])
 
+        choose_number = "ENTER THE NUMBER FOR YOUR CHOICE, OR PRESS <ENTER> TO ACCEPT THE DEFAULT::"
         qa = {
-              'PRESS <ENTER> TO CONTINUE:': '',
-              'DO YOU ACCEPT THE TERMS OF THIS LICENSE AGREEMENT? (Y/N):': 'Y',
-              'ENTER AN ABSOLUTE PATH, OR PRESS <ENTER> TO ACCEPT THE DEFAULT :': self.installdir,
-              'IS THIS CORRECT? (Y/N):': 'Y',
-              'Do you wish to have a shortcut installed on your desktop? ->1- Yes 2- No ENTER THE NUMBER FOR YOUR CHOICE, OR PRESS <ENTER> TO ACCEPT THE DEFAULT::': '2',
-              '->1- Single User License 2- Network License ENTER THE NUMBER FOR YOUR CHOICE, OR PRESS <ENTER> TO ACCEPT THE DEFAULT::': '2',
-              'PRESS <ENTER> TO EXIT THE INSTALLER:': '',
-              'License server (DEFAULT: ):': self.cfg['license_server'],
-              'Port number (optional) (DEFAULT: ):': '',
-              '->1- Configure toolbox for Matlab 2- Do not configure at this time ENTER THE NUMBER FOR YOUR CHOICE, OR PRESS <ENTER> TO ACCEPT THE DEFAULT::': '2'
-             }
-
-        no_qa = ['Graphical installers are not supported by the VM. The console mode will be used instead...',
-                 'Extracting the JRE from the installer archive...',
-                 'Launching installer...',
-                 "Configuring the installer for this system's environment...",
-                 'Unpacking the JRE...',
-                 '\[[-|]*']
+            'PRESS <ENTER> TO CONTINUE:': '',
+            'DO YOU ACCEPT THE TERMS OF THIS LICENSE AGREEMENT? (Y/N):': 'Y',
+            'ENTER AN ABSOLUTE PATH, OR PRESS <ENTER> TO ACCEPT THE DEFAULT :': self.installdir,
+            'IS THIS CORRECT? (Y/N):': 'Y',
+            'Do you wish to have a shortcut installed on your desktop? ->1- Yes 2- No ' + choose_number: '2',
+            '->1- Single User License 2- Network License ' + choose_number: '2',
```

Comment:
we only install with a license server, and trying to select the single user license opens Pandora's box with more Q&A stuff... not going there, since we don't need it (right now)